RecoTauTag/RecoTau/src/DeepTauBase.cc

```diff
@@ -76,17 +76,17 @@ DeepTauBase::Output::ResultMap DeepTauBase::Output::get_value(const edm::Handle<
     return output;
 }
 
-DeepTauBase::DeepTauBase(const edm::ParameterSet& cfg, const OutputCollection& outputCollection, const DeepTauCache* _cache) :
-    taus_token(consumes<TauCollection>(cfg.getParameter<edm::InputTag>("taus"))),
-    outputs(outputCollection),
-    cache(_cache)
+DeepTauBase::DeepTauBase(const edm::ParameterSet& cfg, const OutputCollection& outputCollection, const DeepTauCache* Cache) :
+    tausToken_(consumes<TauCollection>(cfg.getParameter<edm::InputTag>("taus"))),
+    outputs_(outputCollection),
+    cache_(Cache)
 {
-    for(const auto& output_desc : outputs) {
+    for(const auto& output_desc : outputs_) {
         produces<TauDiscriminator>(output_desc.first);
         const auto& cut_pset = cfg.getParameter<edm::ParameterSet>(output_desc.first + "WP");
         for(const std::string& wp_name : cut_pset.getParameterNames()) {
```

Comment:
I do not have a strong preference for naming convention for local variables, but if you want change this it will be good do it consequently, e.g. `wp_name` -> `wpName`; and similarly: `cut_pset`, `result_map`, `output_desc`,  etc.